backend/user/signals.py

```
@@ -1,10 +1,12 @@
+import logging
+
 from django.db.models.signals import post_save
 from django.dispatch import receiver
 
-import logging
-
+from airbnb import cipher
```

Ruff (0.8.2)

`airbnb.cipher` imported but unused

Remove unused import: `airbnb.cipher`

(F401)